kyuubi-server/src/test/scala/org/apache/kyuubi/operation/KyuubiOperationPerConnectionSuite.scala

```diff
@@ -97,4 +97,39 @@ class KyuubiOperationPerConnectionSuite extends WithKyuubiServer with HiveJDBCTe
         "Caused by: java.net.SocketException: Broken pipe (Write failed)")
     }
   }
+
```

Comment:
can we also add a test to test in async mode still work for USER mode to share the same engine？